library/VTKExtensions/Rendering/CMakeLists.txt

```diff
@@ -25,8 +44,13 @@ set(classes
   vtkF3DRendererWithColoring
   )
 
+# Needs https://gitlab.kitware.com/vtk/vtk/-/merge_requests/10675
+if(NOT ANDROID AND NOT EMSCRIPTEN AND VTK_VERSION VERSION_GREATER_EQUAL 9.3.20240203)
+  set(classes ${classes} vtkF3DBitonicSort vtkF3DPointSplatMapper)
```

Comment:
should we exclude that on apple too ?

Reply:
It is checked at runtime with `vtkShader::IsComputeShaderSupported()`

Reply:
Fair enoug. maybe macos will support compute shader in OpenGL at some point ^^

Reply:
I don't think so but who knows